pkg/block/block.go

```diff
@@ -123,6 +123,20 @@ func cleanUp(logger log.Logger, bkt objstore.Bucket, id ulid.ULID, err error) er
 
 // MarkForDeletion creates a file which stores information about when the block was marked for deletion.
 func MarkForDeletion(ctx context.Context, logger log.Logger, bkt objstore.Bucket, id ulid.ULID, markedForDeletion prometheus.Counter) error {
+	return markForDeletion(ctx, logger, bkt, id, time.Now(), markedForDeletion)
```

Comment:
This function looks very shallow to me, maybe we can avoid this and inline, WDYT? (:

Reply:
I don't understand what you mean with inline. 

I did it this way to make it easier to preserve current behaviour of the function and make it backwards compatible. Instead of changing the signature to add an optional time argument.